### PR TITLE
Changes to the theme switch flow

### PR DESCRIPTION
**Pull request by mendezcode**
✓ Updates the theme switch flow to include new wording for the theme activation page:

screen shot 2017-11-13 at 2 25 53 pm

---

✓ Updates theme info page with the following changes:

- Modify "Customize" button label to "Customize site"
- Remove "Open Live Demo" button when viewing info page of active theme

screen shot 2017-11-13 at 2 27 09 pm

---
@mapk feel free to approve after testing if everything has been done according to spec.
@seear Please help us out with the code review + testing of the above.

Thanks in advance comrades!

### CODE REVIEW

File: client/my-sites/themes/thanks-modal.jsx [hunk 1]
```diff
@@ -156,17 +148,17 @@ const ThanksModal = createReactClass( {
 
 	render() {
 		const { currentTheme, hasActivated, isActivating } = this.props;
-		const visitSiteText = hasActivated
-			? translate( 'Visit site' )
+		const customizeSiteText = hasActivated
+			? translate( 'Customize site' )
```
Comment (a8ci18n) [inline client/my-sites/themes/thanks-modal.jsx]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Customize your site' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/theme-switch-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/themes/thanks-modal.jsx [hunk 2]
```diff
@@ -156,17 +148,17 @@ const ThanksModal = createReactClass( {
...
 			: translate( 'Activating theme…' );
 		const buttons = [
-			{ action: 'back', label: translate( 'Back to themes' ), onClick: this.goBack },
+			{ action: 'learn', label: translate( 'Learn this theme' ), onClick: this.learnThisTheme },
```
Comment (a8ci18n) [inline client/my-sites/themes/thanks-modal.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Learn more about this theme' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/themes/thanks-modal.jsx [hunk 3]
```diff
@@ -134,14 +122,18 @@ const ThanksModal = createReactClass( {
 		return (
 			<div>
 				<h1>
-					{ translate( 'Thanks for choosing {{br/}} %(themeName)s {{br/}} by %(themeAuthor)s', {
-						args: { themeName, themeAuthor },
+					{ translate( 'Thanks for choosing {{br/}} %(themeName)s', {
```
Comment (a8ci18n) [inline client/my-sites/themes/thanks-modal.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Thanks for choosing %1$s by %2$s.' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/theme-switch-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by mendezcode** [comment]
Comments above by @a8ci18n are accurate, but irrelevant to this PR, since we're changing those texts.

**Comment by seear** [comment]
screen shot 2017-11-13 at 2 25 53 pm

One small problem with this now is that there is no longer a button to dismiss the dialog.

**Comment by mapk** [comment]
I'm having difficulties testing this out locally (I can never login to calypso.localhost:3000), and the live test branch isn't booting up for me. 

> Just the design question about the lack of dismiss button on the modal, @mapk can give an opinion there.

In regards here, I'm okay without a `dismiss` button. The user can click anywhere outside the modal to close it if they needed to.

**Comment by mapk** [comment]
@mendezcode Can we add the word "about" to the first button in that modal? So it would say, "Learn about this theme".  Thanks!

**Comment by mendezcode** [comment]
@seear, Thanks for the eyes on the code. Is this baby ready to approve then?

File: client/my-sites/theme/main.jsx [hunk 4]
```diff
@@ -453,7 +453,10 @@ class ThemeSheet extends React.Component {
 
 	getDefaultOptionLabel = () => {
 		const { defaultOption, isActive, isLoggedIn, isPremium, isPurchased } = this.props;
-		if ( isLoggedIn && ! isActive ) {
+		if ( isActive ) {
+			// Customize size
+			return i18n.translate( 'Customize site' );
```
Comment (a8ci18n) [inline client/my-sites/theme/main.jsx]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Customize your site' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/theme-switch-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**